app/static/styles/app.css

```diff
@@ -16,7 +16,11 @@ nav .mobile.only.row .ui.vertical.menu {
 
 @media -sass-debug-info{filename{}line{font-family:\0000325}}
 .flashes > .ui.message:last-of-type {
-  margin-bottom: 2rem;
+    float: right;
```

Comment:
You should edit the assets/scss files instead! We have a library that will precompile all the scss to app.css and so we shouldn't be touching styles/css files directly